SofaKernel/modules/SofaHelper/src/sofa/helper/ComponentChange.cpp

```diff
@@ -140,6 +140,18 @@ std::map<std::string, ComponentChange> uncreatableComponents = {
     {"SparseGridMultipleTopology", Pluginized("v20.06", "SofaNonUniformFem")},
     {"SparseGridRamificationTopology", Pluginized("v20.06", "SofaNonUniformFem")},
 
+
+    /***********************/
+    // REMOVED SINCE v20.12
+    { "DynamicSparseGridTopologyAlgorithms", Deprecated("v20.12", "v21.06") },
```

Comment:
Removed vs Deprecated

Reply:
I would like to remove them... do we have to go through the 6 months of deprecated before... ?